info_pages/views.py

```diff
@@ -0,0 +1,6 @@
+from django.shortcuts import render
+
+# Create your views here.
+
+def project(request, project_name):
+    return render(request, 'static/project.html', {'project_name': project_name})
```

Comment:
You are returning 'project_name' variable, without mentioning it in the function. 
The structure needs to be looking like that: 

def project(request, project_id):
    project = Project.objects.get(id=project_id) ##Where 'Project' is a model in database

    context = {'project':project}
    return render(request, 'info_pages/project.html', context) ##also please rename the static folder inside templates into info pages so it looks like templates/info_pages/project.html. The name static is used for CSS and JS files